### ISSUE TITLE
Rumble sometimes turns off the controller

### ISSUE THREAD

**Issue by arthurlutz** [kozec/sc-controller#658]
When using sc-controller (thanks ! It's so awesome!) I sometimes have the controller turn itself off during a game. Investigating the issue I detected that it might be related to "heavy" rumble activity. So I tried to turn off the rumble and the controller stayed on. 

Could it be related to a low battery level ? (Not sure how to check that)  Could it be a software bug ?

**Comment by kozec** [comment]
Yes, almost definitelly. It happens to me as well when batteries are close to empty. Best way to check is to verify it with new / freshly charged batteries.
